config/local.ini

```diff
@@ -122,5 +152,18 @@ args = (sys.stderr,)
 level = NOTSET
 formatter = color
 
+[handler_sentry]
+class = raven.handlers.logging.SentryHandler
+args = ('https://<key>:<secret>@app.getsentry.com/<project>',)
+level = ERROR
+formatter = generic
```

Comment:
Is this something we want enabled locally? Or does it not really matter if someone runs the container locally with this enabled?

Reply:
I hesitated, but thought that it wouldn't harm to leave it.  Alternatively we could have a `prod.ini` file example with that sort of things